File: ishicodes/solutions/2026-06-22/Q103.c
/*
 * Problem: Create ATM simulation.
 */

#include <stdio.h>
int main() {
  int b = 1000, c, a;
  while (1) {
    scanf("%d", &c);
    if (c == 1)
      printf("Balance: %d\n", b);
    else if (c == 2) {
      scanf("%d", &a);
      b += a;
    } else if (c == 3) {
      scanf("%d", &a);
      b -= a;
    } else
      break;
  }
  return 0;
}

File: ishicodes/solutions/2026-06-22/Q103.py
# Problem: Create ATM simulation.

b = 1000
while True:
    c = int(input())
    if c == 1:
        print(b)
    elif c == 2:
        b += int(input())
    elif c == 3:
        b -= int(input())
    else:
        break
